uzdevums_4.py

```diff
@@ -0,0 +1,13 @@
+# Izvadīt uz ekrāna divu no lietotāja saņemto skaitļu:
+# summu
+# reizinājumu
+# dalījumu
+print ("Ievadiet divus skaitlus, lai apreikinatu to summu, reizinajumu un dalijumu : ")
+num_1 = int(input())
+num_2 = int(input())
+product_1 = num_1 + num_2
+print ("Summa ir : ", product_1)
+product_2 = num_1 * num_2
+print ("Reizinajums ir : ", product_2)
+product_3 = num_1 * num_2
```

Comment:
Izkatās pēc copy/paste kļūdas darbībā:
```suggestion
product_3 = num_1 / num_2
```